### PR TITLE
removed verification link and related comments for the change user em…

### PR DESCRIPTION
**Pull request by snopeks**
We are removing email verification from the change-user-email flow. I've minimally edited the emails to account for this change.

### CODE REVIEW

**Comment by stubbornella** [comment]
Looks good to me from a product perspective. Let's have @jefflembeck check everything out re: handlebars vars.

File: revert-email-change.text.hbs [hunk 1]
```diff
@@ -3,13 +3,13 @@ You are receiving this because you (or someone else) requested that the email ad
 old address: <{{changeEmailFrom}}>
 new address: <{{changeEmailTo}}>
 
-If this was intentional, you can safely ignore this message. However, a confirmation email was sent to <{{changeEmailTo}}> with a link that must be clicked to complete the process.
+If this was intentional, you can safely ignore this message.
 
 IMPORTANT: If this was NOT intentional, then your account MAY have been compromised.  Please click the following link to revert the change immediately:
 
 {{host}}/email-edit/revert/{{{tokenHelper name=name changeEmailFrom=changeEmailFrom changeEmailTo=changeEmailTo token=revToken confToken=confToken prefix="email_change_rev_"}}}
```
Comment (jefflembeck) [inline revert-email-change.text.hbs]: You'll want to add a ttl variable here so you can guarantee the link will expire in 7 days

```
{{! 604800 = one week }}
{{host}}/email-edit/revert/{{{tokenHelper name=name changeEmailFrom=changeEmailFrom changeEmailTo=changeEmailTo token=revToken prefix="email_change_rev_" ttl=604800}}}
```

Comment (stubbornella) [inline revert-email-change.text.hbs]: oh cool! I didn't know that was how it worked! Neato.

**Comment by jefflembeck** [comment]
🐑

**Pull request merged**